### DISCUSSION TITLE
[Feature] Option to hide GPS data on shared images

### DISCUSSION THREAD

**Discussion by mattinx** [immich-app/immich#9835] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When sharing an image or album with a link to be posted in a public location, it is often desirable to strip the GPS (and potentially other identifying metadata) from the images available through the link. For example, if you don't want to publicly share your home or office location. Google photos does this automatically when generating a sharing link, as does Imgur when posting images.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bkidwell** [comment]
Additionally, when this feature has been implemented in Immich's server, I feel that it should be enabled by default globally or per account, until the user decides to make their account default to "share GPS data when publicly sharing a photo" instead.

I think most users would not want to leak this data in public, and are unlikely to go check to see if it is shared.

**Comment by mmomjian** [comment]
We support this, if you disable the "download" and "show metadata" options in an album.

**Reply by TodePond** [reply]
We're currently setting up immich for a community group and we'd like to allow downloads but strip location data on upload. Do you have any recommendations for this?

**Reply by mFIND** [reply]
I second this. I also would like to be able to share album and allow friends & family to download images directly, but not necessarily let them see metadata outright (they won't use it anyways)

I wouldn't even mind, if public users were allowed to see the map, but upon downloading, the metadata was stripped.
